bigquery_etl/glean_usage/common.py

```diff
@@ -256,8 +256,8 @@ def generate_per_app(self, project_id, app_info, output_dir=None):
                 return
 
             if output_dir:
-                write_dataset_metadata(output_dir, table)
+
                 write_sql(output_dir, table, "query.sql", query_sql)
                 write_sql(output_dir, table, "metadata.yaml", metadata)
                 write_sql(output_dir, view, "view.sql", view_sql)
-                write_dataset_metadata(output_dir, table)
+                write_dataset_metadata(output_dir, table, public_facing_only=False)
```

Comment:
Currently, the `dataset-metadata.yaml` file isn't added to `_derived` dataset directories.